telco-core/configuration/reference-crs/optional/other/server-cert-openshift-config-copy.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: cluster.open-cluster-management.io/v1beta2
+kind: ManagedClusterSetBinding
+metadata:
+  name: global
+  namespace: openshift-config
+spec:
+  clusterSet: global
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: copy-config-secrets
```

Comment:
Maybe be a bit more specific so it is clear when users see it in the ACM UI
```suggestion
  name: copy-platform-route-managed-secrets
```

Reply:
Done.  Thanks!